test/Cardano/FeeSpec.hs

```diff
@@ -712,6 +721,79 @@ propRebalanceChangeOutputsPreservesSum
         | otherwise =
             null coinsRemaining
 
+{-------------------------------------------------------------------------------
+                           splitCoin - Properties
+-------------------------------------------------------------------------------}
+
+data SplitCoinData = SplitCoinData
+    { scdCoinToSplit :: Coin
+    , scdCoinsToIncrease :: [Coin]
+    } deriving (Eq, Generic, Show)
+
+instance Arbitrary SplitCoinData where
+    arbitrary = do
+        coinToSplit <- genCoin
+        n <- oneof
+            [ pure 0
+            , pure 1
+            , choose (2, 10)
+            ]
+        coinsToIncrease <- replicateM n genCoin
+        pure $ SplitCoinData coinToSplit coinsToIncrease
+      where
+        genCoin :: Gen Coin
+        genCoin = oneof
+            [ pure minBound
+            , pure maxBound
+            , Coin <$> choose
+                ( succ . getCoin $ minBound
+                , pred . getCoin $ maxBound
+                )
+            ]
+
+propSplitCoinDataCoverage :: SplitCoinData -> Property
+propSplitCoinDataCoverage (SplitCoinData coinToSplit coinsToIncrease) =
```

Comment:
what about a situation when we have `cointToSplit = N` and the number of coins to be increased more than `N` ?

Reply:
probably this coverage is missing ^^^

Reply:
Fixed in the following commits:
3c3493b16b1d814e892e9a491b735fad3bddf5f5
90b7dbf9614116a00e57c9136d4b773c51235a05